core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -49,12 +49,13 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, scheduler: TaskSchedule
 
   // executor ID -> timestamp of when the last heartbeat from this executor was received
   private val executorLastSeen = new mutable.HashMap[String, Long]
-  
-  private val executorTimeoutMs = sc.conf.getLong("spark.network.timeout", 
-    sc.conf.getLong("spark.storage.blockManagerSlaveTimeoutMs", 120)) * 1000
-  
-  private val checkTimeoutIntervalMs = sc.conf.getLong("spark.network.timeoutInterval",
-    sc.conf.getLong("spark.storage.blockManagerTimeoutIntervalMs", 60)) * 1000
+
+  private val executorTimeoutMs = sc.conf.getOption("spark.network.timeout").map(_.toLong * 1000).
```

Comment:
You might comment that `timeout` is in seconds and (obviously) `blockManagerSlaveTimeoutMs` is milliseconds. I agree with the change though. These properties you're fixing aren't documented right? the only ref I saw on the mailing list clearly directed people to set values like "60000", which is correctly in ms.